spec/hosts/nodes/auxtel-hcu02.cp.lsst.org_spec.rb

```diff
@@ -28,37 +28,41 @@
       it { is_expected.to compile.with_all_deps }
 
       include_examples 'baremetal no bmc'
-      include_context 'with nm interface'
-      it { is_expected.to have_nm__connection_resource_count(4) }
```

Comment:
Having a resource count for `network__interface` is a good idea to make sure extra interfaces aren't being created by accident.